Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/Utilities/WeakEventListener.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Shouldn't this be added to the Microsoft.Toolkit.UWP Library?

Reply:
We wanted Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.dll to be a standalone DLL with no dependencies on other toolkit DLLs whatsover.